src/component/Button.js

```diff
@@ -1,14 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import '../index.scss';
 
-const Button = ({ name }) => <div>{name}</div>;
+const Button = ({ name, wide, color }) => {
+  const buttonPropStyle = {
```

Comment:
- [ ] **[OPTIONAL]** It's often recommended to use css classes for styling instead of inline styles. Check out [this](https://reactjs.org/docs/faq-styling.html#are-inline-styles-bad) from react docs.

Reply:
Thanks for your feedback. I will update _wide_. As I know, there is no way to pass a parameter to CSS classes. So, inline styles must be used to change the background color by using any color. If I have any mistake, let me know it, please.

Reply:
You can use conditional CSS styles as follows:
```js
<button className={` ${wide ? "btn--wide" : "btn--normal"} }>
```

Please check out [this](https://www.pluralsight.com/guides/applying-classes-conditionally-react) for more.

Reply:
Thanks for your suggestion. I updated _wide_ by using your suggestion. But, there is no predefined colors for background. For example, "#999" or "red" or "#222" or "black" cannot be used by using external css if all classes aren't added one by one. So, I couldn't update _color_ by using css classes. If I have a mistake, let me know it, please.

Reply:
Not sure that I fully understand you, but here is my suggestion:

```js
const Button = ({ name, wide, color }) => (
  <div className={`button ${wide ? 'btn--wide' : 'btn--normal'} ${color === 'orange' ? 'btn--orange' : 'btn--white'}`}>
    {name}
  </div>
);
```

and then in your css you will color classes:
```css
.btn--orange {
  background-color: orange;
}

.btn--white {
  background-color: white;
}
```

Please let me know if this is what you meant

Reply:
In your suggestion, there are two different color options; white or orange. In this situation, classes can be used. 
In your suggestion, if color prop is used as "red", there should be a class for "red" color, but it is not defined. So, the background color won't be shown as red. 
![image](https://user-images.githubusercontent.com/44648011/98082445-ca668000-1e89-11eb-8887-7776a0947fd6.png)
But, there is no predefined colors for this project.
```JavaScript
import React from 'react';
const Button = ({ color }) => { 
const buttonPropStyle = {
    backgroundColor: color,
  };

return (
    <button type="button" style={buttonPropStyle}>
    </button>
  );
```
If inline style is used as above, color prop can be anything like "white","red","#333","rgb(5,5,5)". 
Is it clear? If not, let me know it, please.